go.mod

```diff
@@ -3,7 +3,7 @@ go 1.21
 module github.com/cosmos/ibc-go/v8
 
 require (
-	cosmossdk.io/api v0.7.2
+	cosmossdk.io/api v0.7.3-0.20231029200940-6af7f30bfd54
```

Comment:
what is this change for? I see v0.50.1, uses v0.7.2 https://github.com/cosmos/cosmos-sdk/blob/v0.50.1/go.mod#L8

Reply:
I don't know, but I downgraded it v7.2.0 now.